src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx

```diff
@@ -183,7 +183,7 @@
     <value>Creates a directory object from a path</value>
   </data>
   <data name="DirectoryObjectPortDataPathToolTip" xml:space="preserve">
-    <value>Path to the directory.</value>
+    <value>Path to the directory. (string)</value>
```

Comment:
Still really dont like this... Is this something you agreed with ADSK?

Reply:
the comment on this one was to make sure that the user knows the datatype needed for the input `path`
Im changing this to the style that was suggested for other nodes. `type: string (Path to the directory)`
in 01eb3dd